web/app/js/components/PodDetail.jsx

```diff
@@ -99,15 +99,15 @@ export default class PodDetail extends React.Component {
     let currentSuccessRate = _.get(_.last(_.get(this.state.podTs, "SUCCESS_RATE", [])), "value");
 
     return [
-      <HealthPane
+      <ResourceHealthOverview
         key="pod-health-pane"
         entity={this.state.pod}
         entityType="pod"
         currentSr={currentSuccessRate}
         upstreamMetrics={this.state.upstreamMetrics}
         downstreamMetrics={this.state.downstreamMetrics} />,
       _.isEmpty(this.state.podTs) ? null :
-        <StatPane
+        <ResourceMetricsOverview
           key="pod-stat-pane"
           lastUpdated={this.state.lastUpdated}
           timeseries={this.state.podTs} />,
```

Comment:
just below this, we didn't seem to have changed `entity` to `Resource` in the `UpstreamDownstream` component props as we did in `DeploymentDetail`, as well as the any code changes in the `UpstreamDownstream` component and the encompassing `TabbedMetricsTable` component.